content/800-data-platform/05-data-browser.mdx

```diff
@@ -0,0 +1,165 @@
+
+You can also discard accumulated changes with **Discard changes**.
+
+### Add a record
+
+1. In the model view, click **Add record**.
+2. Based on the data allowed in each field, type the data for the record.
+
+   | Field data type | Description                                                                                                                                                                                                                                                         |
+   | :--------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+   |       `A?`       | Text                                                                                                                                                                                                                                                                |
+   |       `#`        | Integer<br /><br />If such a field has `autoincrement()` pre-filled, do not edit the cell and add a number.                                                                                                                                                        |
+   |       `[]`       | List<br /><br />It typically referes to a list of records that exist in another model in the database.                                                                                                                                                                  |
+   |       `{}`       | Relation with a model defined separately in the database.<br /><br />Typically, you need to select the same value as any of the previous records<br />Click the name of the model to see the list of values which you can then select for the related field. |
+
+3. (Optional) If you are unhappy with your changes, click **Discard changes** and start over.
+4. Click **Save 1 change**.
+
+### Edit a record
+
+1. Double-click in a cell with existing date to edit its contents.
+2. (Optional) If you are unhappy with your changes, click **Discard changes** and start over.
+3. Click **Save 1 change**.
+
+### Delete a record
+
+1. From the left column, select the check box for the record you want to delete.
+2. Click **Delete 1 record**.
+3. Click **Save 1 change**.
+
+### Edit multiple records at once
+
+You can add multiple records and edited cells and, thus, accumulate them.
+
+In the end, click **Save&nbsp;<number\>&nbsp;changes** to finalize them.
+
+<Admonition type="warning">
+
+**Warning**<br /><br />
+Deleting a record is a separate operation that force-saves any unsaved edits and is then completed.
```

Comment:
Bug or feature?

Reply:
Not sure. It is the current behavior. I've edited the text to hopefully make it clearer. I'll track this conversation with Spiros to figure out if it's a bug.